sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -48,3 +45,5 @@
 =sys-libs/liburing-2.1-r2 ~amd64 ~arm64
 
 =app-crypt/adcli-0.9.2 ~amd64 ~arm64
+=sys-apps/nvme-cli-2.4-r2 ~amd64 ~arm64
+=sys-libs/libnvme-1.4 ~amd64 ~arm64
```

Comment:
These are not necessary any more - these ebuilds got stabilized on both amd64 and arm64 since I made a comment about these.